channels/OFF-CHAIN.md

```diff
@@ -0,0 +1,503 @@
+| minimum_depth        | 4  |
+ ---------------------- ----
+| initiator_amount     | 8  |
+ ---------------------- ----
+| responder_amount     | 8  |
+ ---------------------- ----
+| channel_reserve      | 8  |
+ ---------------------- ----
+| responder_pubkey     | 32 |
+ ---------------------- ----
+```
+
+- `chain_hash`: transaction hash of the chain you want to use
+- `temporary_channel_id`: randomly chosen id unique between the involved peers,
+- `minimum_depth`: number of blocks until an opening transaction should be
+  considered final. The `minimum_depth` is set by the responding peer, since
+  they will typically be the one providing a service.
+- `initiator_amount`: amount the initiator is willing to commit
+- `responder_amount`: amount the initiator wants the responder to commit
+- `channel_reserve`: the minimum amount both peers need to maintain. This makes
+  sure that both have to lose something in case they act maliciously
+- `responder_pubkey`: the account that the initiator wants to use to open the
+  channel
+
+(***TODO***: This could be interactive, i.e. if the responding peer sends
+different amounts, then that might communicate that it wants these instead.)
+
+
+#### Requirements
+
+*Responder*:
+
+- `chain_hash` MUST identify the chain to be used
+- `temporary_channel_id` MUST be unique between the involved peers
+- `lock_period` SHOULD be sufficient time to safely publish transactions to the
+  blockchain to stop a cheater
+- `responder_pubkey` MUST be a valid ed25519 pubkey
+
+
+### `funding_created`
+
+In order to open a channel on chain, peers need to cooperate and co-sign the
+`channel_create` transaction. The `funding_created` message is used by the
+initiator to send their signature to the responder.
+
+```
+  name                  size (bytes)
+ ---------------------- ----
+| temporary_channel_id | 32 |
+ ---------------------- ----
+| transaction_data     | 32 |
+ ---------------------- ----
+| signature            | 64 |
+ ---------------------- ----
+```
+
+
+#### Requirements
+
+*Responder*:
+
+- MUST abort if the signature is invalid for the provided transaction data
+
+
+### `funding_signed`
+
+If the responder was able to validate the initiator's signature sent in the
+`funding_created` message, then it should proceed to respond with its
+`funding_signed` message, unless the initiator specified a `push_amount > 0` in
+the `channel_open` message. In the latter case, they should wait for the
+initiator to send the initial state signed by them.
+
+
+```
+  name                  size (bytes)
+ ---------------------- ----
+| temporary_channel_id | 32 |
+ ---------------------- ----
+| signature            | 64 |
+ ---------------------- ----
+```
+
+
+### `funding_locked`
+
+Opening a channel requires an on-chain transaction. This transaction needs to be
+included in a block and, since we only have probabilistic finality, be
+sufficiently confirmed, s.t. the probability of a chain re-organisation is
```

Comment:
"s.t." = "so that"? I find this reduces readability. Though totally optional of course.